qa-portal-angular/projects/qa-cv/src/app/view-cv/view-cv.component.ts

```diff
@@ -1,21 +1,21 @@
 import {Component, OnInit, Output} from '@angular/core';
```

Comment:
Isn't this file redundant now?

Reply:
qa-portal-angular/projects/qa-cv/src/app/view-cv/view-cv.component.html
is deleted on this branch as it is not needed.
I don't understand why this has appeared in the merge.